tests/test_section.py

```diff
@@ -111,6 +125,22 @@ def it_can_change_its_page_margins(self, margins_set_fixture):
 
     # fixtures -------------------------------------------------------
 
+    @pytest.fixture
+    def Header_(self, request, header_):
+        return class_mock(
+            request, 'docx.section.Header',
+            return_value=header_
+        )
+
+    @pytest.fixture
+    def header_(self, request):
+        return instance_mock(request, Header)
```

Comment:
These two (Header_ and header_) go in a separate subsection (immediately following the fixtures) entitled:

```python
# fixture components -----------------
```

If you have a look around at the other test classes you soon find an example you can cut and paste. The basic notion is that these are simple, atomic mocks that can be reused in multiple unit tests. Generally there is one fixture per test, potentially multiple components per fixture. I sense you've already worked this out, but the trailing underscore indicates it's a mock. It helps keep track of what's real and what's mock when they're put to use in the test itself.